src/models/vertexai.rs

```diff
@@ -0,0 +1,250 @@
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+use crate::models::chat::{ChatCompletion, ChatCompletionChoice, ChatCompletionRequest};
+use crate::models::content::{ChatCompletionMessage, ChatMessageContent};
+use crate::models::tool_calls::{ChatMessageToolCall, FunctionCall};
+use crate::models::tool_choice::{SimpleToolChoice, ToolChoice};
+use crate::models::usage::Usage;
+
+#[derive(Debug, Serialize, Deserialize)]
```

Comment:
Please follow the similar structure we have in other providers - this should be under providers/vertexai/models.rs